### PR TITLE
Style fix

### PR DESCRIPTION
**Pull request by Sarahshr**
Rework style and delete arrays' files

### CODE REVIEW

**Review by ThomasMlh — DISMISSED** [review]
It's ok for me ! 
It's mostly minors changes and a lot of jpg/png deletions

File: src/styled-components/Chapters.js [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
-  font-size: 15pt;
-`;
-
-export const Schema = styled.img`
-  margin: 4vh;
-`;
-
-export const SchemaContainer = styled.div`
+export const SectionContainer = styled.div`
+  //border: 1px solid blue;
```
Comment (LoicBrassart) [inline src/styled-components/Chapters.js]: Dead code

File: src/styled-components/Chapters.js [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
   display: flex;
+  align-items: center;
+  flex-direction: column;
   justify-content: center;
+  //padding: 0 5%;
```
Comment (LoicBrassart) [inline src/styled-components/Chapters.js]: Dead code

File: src/styled-components/Chapters.js [hunk 3]
```diff
@@ -1,60 +1,50 @@
...
-    display: none;
-  }
-  &#hideShortcut {
-    translate: -8vw;
-    transition: all 0.5s ease-in;
-  }
-  &#showShortcut {
-    transition: all 0.5s ease-in;
+export const Paragraph = styled.p`
+  //border: 1px solid red;
```
Comment (LoicBrassart) [inline src/styled-components/Chapters.js]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A few lines of dead code

**Review by ThomasMlh — APPROVED** [review]
OK !

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**